pkg/contract_watcher/repository/header_repository_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/makerdao/vulcanizedb/pkg/contract_watcher/helpers/test_helpers/mocks"
 	"github.com/makerdao/vulcanizedb/pkg/contract_watcher/repository"
 	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore"
```

Comment:
How come we're importing the HeaderRepository as `datastore` now instead of `repositories`? It looks like it's still in the `repositories` package.

Reply:
`datastore` is the interface, `repositories` is the (now private) implementation